test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,41 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list (it has no null objects)
+        assertHasNoNull();
+
+        // there are null objects
+        assertHasNull((Object) null);
+        assertHasNull((Object) null, 1);
+        assertHasNull((Object) null, "");
+        assertHasNull((Object) null, "abc");
+        assertHasNull((Object) null, 1, "", "abc");
+
+        // there are no null objects
+        assertHasNoNull(objectList);
+        assertHasNoNull(1, 2);
+        assertHasNoNull("abc", "ab", "a");
+        assertHasNoNull(objectList, "def", "ghi", "jkl");
+
+        // some null objects
+        assertHasNoNull("abc", "abc");
+        assertHasNoNull(1, Integer.valueOf(1));
+        assertHasNoNull("", "abc", "a", "abc");
+
+        assertHasNull(null, null);
+        assertHasNull(null, 1, Integer.valueOf(1));
+        assertHasNull(null, "a", "b", null);
+        assertHasNull(objectList, null, "mno", 3, "pqr", "x");
+    }
+
+    private void assertHasNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertHasNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```

Comment:
sorry, this one was from previous LO

Reply:
Please refrain from adding unrelated code in the future PRs